lib/gis/file_name.c

```diff
@@ -27,8 +27,29 @@ static void append_char(char*, char);
   If <i>name</i> is of the form "nnn@ppp" then path is set as if name
   had been "nnn" and mapset had been "ppp" (mapset parameter itself is
   ignored in this case).
-  
-  \param[out] path buffer to hold resultant full path to file
+
+  Paths to files are in form:
+  /home/user/GISDBASE/LOCATION/mapset/element/name
+
+  path input buffer memory must be preallocated by caller.
```

Comment:
"must be allocated by caller" and "path is a preallocated buffer". I think _preallocating_ raises an unnecessary question: is is same as allocating?

Reply:
Then plain allocating it is.